docs/http/migration-guide.md

```diff
@@ -0,0 +1,212 @@
+# HTTP semantic convention stability migration guide
+
+Due to the significant number of modifications and the extensive user base
+affected by them, existing HTTP instrumentations published by
+OpenTelemetry are required to implement a migration plan that will assist users in
+transitioning to the stable HTTP semantic conventions.
+
+Specifically, when existing HTTP instrumentations published by OpenTelemetry are
+updated to the stable HTTP semantic conventions, they:
+
+- Need to introduce an environment variable `OTEL_SEMCONV_STABILITY_OPT_IN` in
```

Comment:
Is `Need` the proper "level" here?  I thought we used `SHOULD` for the env var.

Reply:
Yep, much as I would like this to be a requirement, I was also under the impression that it was optional.

Reply:
```suggestion
- SHOULD introduce an environment variable `OTEL_SEMCONV_STABILITY_OPT_IN` in
```